src/statement.js

```diff
@@ -41,7 +41,7 @@ pp.isLet = function() {
     let pos = next + 1
     while (isIdentifierChar(this.input.charCodeAt(pos), true)) ++pos
     let ident = this.input.slice(next, pos)
-    if (!this.isKeyword(ident)) return true
+    if (["instanceof", "in"].indexOf(ident) === -1) return true
```

Comment:
Can you lift the array out of the function in order to reuse it? (Or show that major engines optimize it out.)

Reply:
Sure, should that be in `identifier.js`?

Reply:
Yeah, that sounds like a good place.